app/helpers/govspeak_helper.rb

```diff
@@ -263,6 +264,18 @@ def govspeak_with_attachments_and_alt_format_information(govspeak, attachments =
     end
   end
 
+  def convert_brexit_cta(govspeak)
+    return govspeak if govspeak.blank?
+
+    govspeak.gsub(/\$BrexitCTA/) do
+      "$CTA\n" +
```

Comment:
Could we put the html content in a separate file? That would make it easier to find and edit at a later date. 

Not sure where to put it - I was thinking maybe we could render it as normal view template? Let me know what you think

Reply:
Sure - there is some precedent for the helpers to render a HTML view.  We could probably create a markdown view?

Reply:
ah yes of course its markdown - hadn't had my coffee yet. Maybe a markdown view? If you can think of a better place to put it then that would be great